### DISCUSSION TITLE
Have separate Analysts only see sensor data from a specific sensor

### DISCUSSION THREAD

**Discussion by Hotdog2022** [Security-Onion-Solutions/securityonion#3890] [Unsupported Versions]
We're deploying SO with multiple sensors/locations. I only want Analysts to see their data from only their sensor/location but have all sensors forward data to the head office Master server. Is it possible to restrict Analyst view to only see their sensor data?

**Comment by jertel** [comment]
It's not possible today, but the SO team has been considering support for this concept.
